make/Makefile.build.mk

```
@@ -35,6 +35,10 @@ build: go-check
 build-ui:
 	@cd ${ROOTDIR}/frontend && yarn install && yarn run build
 
+## build-ui-test: Runs the yarn commands to build the dev frontend UI
+build-ui-test:
+	@cd ${ROOTDIR}/frontend && yarn && yarn build && yarn test
```

perform the same commands as `build-ui` target for consistency (i.e. explicitly specify `install` and use `run`)
```suggestion
	@cd ${ROOTDIR}/frontend && yarn install && yarn run build && yarn run test
```

---

As an alternative, you could have done this, I dont know what is better:
```
make build-ui-test: build-ui
	@cd ${ROOTDIR}/frontend && yarn run test
```